query-engine/driver-adapters/js/adapter-neon/src/conversion.ts

```diff
@@ -1,4 +1,4 @@
-import { ColumnTypeEnum, type ColumnType } from '@jkomyno/prisma-js-connector-utils'
+import { ColumnTypeEnum, type ColumnType } from '@jkomyno/prisma-adapter-utils'
```

Comment:
Should this have a different name?
```suggestion
import { ColumnTypeEnum, type ColumnType } from '@jkomyno/prisma-driver-adapter-utils'
```

Reply:
Didn't recall seeing `@jkomyno/prisma-driver-adapter-utils`, but sure, I agree

Reply:
I have not defined that as it is internal.